xml/snmp_sysdescr.xml

```diff
@@ -5452,16 +5452,18 @@ Copyright (c) 1995-2005 by Cisco Systems
   <!--======================================================================
                               SonicWall
    =======================================================================-->
-  <fingerprint pattern="^SonicWALL (.*?)\s+\(SonicOS \S+ (\d[^\)]+)\)\s*$">
+  <fingerprint pattern="^SonicWALL (\S+)\s(\d+).*?\(SonicOS \S+ (\d[^\)]+)\)\s*$">
```

Comment:
Did we change the `\s+` to `\s` for a reason?

Reply:
Nope, should be `\s+` good catch